src/Spatial.Tests/Euclidean/CoordinateSystemTest.cs

```diff
@@ -82,6 +82,12 @@ public void RotationAroundVector(string ps, string @as, string vs, string eps)
         [TestCase("0°", "-90°", "0°", "1, 2, 3", "-3, 2, 1")]
         [TestCase("0°", "0°", "90°", "1, 2, 3", "1, -3, 2")]
         [TestCase("0°", "0°", "-90°", "1, 2, 3", "1, 3, -2")]
+        [TestCase("90°", "90°", "90°", "1, 2, 3", "3, 2, -1")]
+        [TestCase("90°", "0°", "90°", "1, 2, 3", "3, 1, 2")]
+        [TestCase("180°", "0°", "90°", "1, 2, 3", "-1, 3, 2")]
+        [TestCase("180°", "-90°", "0°", "1, 2, 3", "3, -2, 1")]
+        [TestCase("90°", "10°", "0°", "1, 2, 3", "-2, 1.506, 2.781")]
+        [TestCase("90°", "10°", "30°", "1, 2, 3", "-0.232, 1.609, 3.370")]
```

Comment:
I would another case, for which no angle is 90° or 180°, just for the sake of completeness (eg. 15°, -23°, 48°).